SlackBotPrototype/DBPreparer/locations-seed.json

```diff
@@ -0,0 +1,9002 @@
+[

```

Comment:
I initially started out with just three cities but felt lame when playing around with it. I considered hooking up an actual service, decided against it and looked for "major cities united states coordinates". The data source I ended up with was a public gist (referenced in the README): https://gist.github.com/Miserlou/c5cd8364bf9b2420bb29 . I wrote a quick script to read it in, map to my model, add the abbreviations, and serialize to this file.

The original data source didn't mention if these were centroids or not. Since you prompted, I plugged a few samples into Google and they look to be centroids. We could cross reference the entries against another data source that are known centroids to be certain.